src/Ztm.Data.Entity.Postgres/ByteaHandler.cs

```diff
@@ -1,66 +1,34 @@
 using System;
-using System.Text;
+using System.Threading.Tasks;
 using NBitcoin;
 using Npgsql;
 using Npgsql.BackendMessages;
 using Npgsql.TypeHandling;
 
 namespace Ztm.Data.Entity.Postgres
 {
-    class ByteaHandler :  NpgsqlSimpleTypeHandlerWithPsv<byte[], uint256>
+    class ByteaHandler : Npgsql.TypeHandlers.ByteaHandler, INpgsqlTypeHandler<uint256>
     {
-        internal readonly bool storeUInt256AsLE;
-
-        public ByteaHandler(bool lendian = false)
-        {
-            storeUInt256AsLE = lendian;
-        }
-
-        #region Read
-        public override byte[] Read(NpgsqlReadBuffer buf, int len, FieldDescription fieldDescription = null)
+        async ValueTask<uint256> INpgsqlTypeHandler<uint256>.Read(NpgsqlReadBuffer buf, int len, bool async, FieldDescription fieldDescription)
         {
             try
             {
                 var bytes = new byte[len];
-                buf.ReadBytes(bytes, 0, len);
-                return bytes;
+                await buf.ReadBytes(bytes, 0, len, async);
+                return new uint256(bytes, false);
             }
             catch (Exception e)
             {
                 throw new NpgsqlSafeReadException(e);
             }
         }
 
-        protected override uint256 ReadPsv(NpgsqlReadBuffer buf, int len, FieldDescription fieldDescription = null)
+        public async Task Write(uint256 value, NpgsqlWriteBuffer buf, NpgsqlLengthCache lengthCache, NpgsqlParameter parameter, bool async)
         {
-            try
-            {
-                var bytes = new byte[len];
-                buf.ReadBytes(bytes, 0, len);
-                return new uint256(bytes, storeUInt256AsLE);
-            }
-            catch (Exception e)
-            {
-                throw new NpgsqlSafeReadException(e);
-            }
+            await buf.WriteBytesRaw(value.ToBytes(false), async);
```

Comment:
`value.ToBytes(false)` is not correct since currently we store it as little-endian.